README.md

```diff
@@ -1,3 +1,20 @@
-##### cluster-policy-controller
+## cluster-policy-controller
+The cluster-policy-controller is responsible for maintaining policy resources necessary to create pods in a cluster. 
+Controllers managed by cluster-policy-controller are:
+* cluster quota reconcilion - manages cluster quota usage
+* namespace SCC allocation controller - allocates UIDs and SELinux labels for namespaces     
 
-TODO: Add content
+The `cluster-policy-controller` runs as a container in the `openshift-kube-controller-manager namespace`, in the kube-controller-manager static pod.
+This pod is defined and managed by the `kube-controller-manager`
```

Comment:
kube-controller-manager-operator, also add link to https://github.com/openshift/cluster-kube-controller-manager-operator/

Reply:
added, thanks